### PR TITLE
[16.0][MIG] account_move_line_sale_info: Migration to 16.0

### PR DESCRIPTION
**Pull request by AlexPForgeFlow**

### CODE REVIEW

File: account_move_line_sale_info/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 ForgeFlow S.L.
```
Comment (mariadforgeflow) [inline account_move_line_sale_info/__manifest__.py]:
```suggestion
# Copyright 2023 ForgeFlow S.L.
```
Just a minor comment, I think it appears in other files as well

**Review by mariadforgeflow — APPROVED** [review]
LGTM!

**Review by JordiMForgeFlow — APPROVED** [review]
Code + functional review

File: account_move_line_sale_info/tests/test_account_move_line_sale_info.py [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+                "company_id": company.id,
+                "company_ids": [(4, company.id)],
+                "groups_id": [(6, 0, group_ids)],
+            }
+        )
+        return user.id
+
+    # def _create_account_type(self, name, atype):
+    #     acc_type = self.acc_type_model.create(
+    #         {"name": name, "type": atype, "internal_group": name}
```
Comment (etobella) [inline account_move_line_sale_info/tests/test_account_move_line_sale_info.py]: Remove this if it is not used anymore.

**Review by etobella — COMMENTED** [review]
Can you remove the configure file? It is empty

**Review by AaronHForgeFlow — APPROVED** [review]
Functional tests

**Review by BernatPForgeFlow — APPROVED** [review]
LGTM!

File: account_move_line_sale_info/__manifest__.py (hunk 1 above)
Comment (AaronHForgeFlow) [reply account_move_line_sale_info/__manifest__.py]: @AlexPForgeFlow  you can use also # Copyright 2020-23 ForgeFlow S.L.

**Review by AaronHForgeFlow — APPROVED** [review]

**Pull request merged**